base/doc/ltnews41.tex

```diff
@@ -355,6 +355,19 @@ \subsection{Promoting PDF 2.0}
 \cs{DocumentMetadata} is used. A different PDF version can be set with
 the \texttt{pdfversion} key.
 
+\subsection{Extended support for pictures}
+
+\emph{todo: write more about the user options}
+
+The tagging of graphics has been reimplemented and now uses
+tagging sockets.  The options have been extended to allow document authors to choose between four
+tagging variants: as illustrative figures, as artifacts (decorations), as replacement for symbols, as normal text (for example todo notes).
+
+The code supports also graphics made with the 
+\pkg{tikz} packages and todo notes made with the \pkg{todonotes} package.
```

Comment:
Ti\textit{k}Z I think

Reply:
the system yes perhaps, but the package (as loaded by `\usepackage`) not. So I would leave it lowercase.